GS01SUMO/GS01SUMO_R00002_ingest.csv

```diff
@@ -46,8 +46,11 @@ mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/om
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl11/imm/ctdbp02/ctdbp02*.DAT,GS01SUMO-RII11-02-DOSTAD032,recovered_host,Available,
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/instruments/dcl11/imm/CTDBP_sn16-50125/SBE*.hex,GS01SUMO-RII11-02-DOSTAD033,recovered_inst,Available,
 mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/cg_data/dcl11/imm/ctdbp03/ctdbp03*.DAT,GS01SUMO-RII11-02-DOSTAD033,recovered_host,Available,
+#mi.dataset.driver.flord_g.ctdbp_p.flord_g_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00002/instruments/dcl11/imm/CTDBP_sn16-50077/SBE*.hex,GS01SUMO-RII11-02-FLORDG031,recovered_inst,Not Expected,No data available

```

Comment:
update looks good - Confirmed via data recovery checklist that instrument not recovered